src/client/App.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import AppRouter from './router/AppRouter';
+
+const App = (props) => (
```

Comment:
What do you think if we keep `class` based component rather than `functional` ?

Reply:
We can. Would You like me to refactor App for class-based?

Reply:
yeah that would be great. 👍

Reply:
Done : ]